docs/DEVELOPING.md

```diff
@@ -111,6 +65,40 @@ cp -r .devcontainer.example .devcontainer
 
 The Development Container configuration mixes [Docker in Docker](https://github.com/microsoft/vscode-dev-containers/tree/master/containers/docker-in-docker) and [C# (.NET)](https://github.com/microsoft/vscode-dev-containers/tree/master/containers/dotnet) definitions. Thanks to it you can use `docker` and `docker-compose` inside the container.
 
+## Building from a command line
+
+This repository uses [Nuke](https://nuke.build/) for build automation.
+
+Support plugins are available for:
+   - JetBrains ReSharper        https://nuke.build/resharper
+   - JetBrains Rider            https://nuke.build/rider
+   - Microsoft VisualStudio     https://nuke.build/visualstudio
+   - Microsoft VSCode           https://nuke.build/vscode
+
+Restore dotnet tools to prepare build tools for solution. This will install dotnet nuke tool locally.
+
+```cmd
+dotnet tool restore
+```
+
+To see a list of possible targets and configurations run:
+
+```cmd
+dotnet nuke --help
+```
+
+To build using default target run:
+
+```cmd
+dotnet nuke
+```
+
+To build using specific target run:
+
+```cmd
+dotnet nuke --target TargetNameHere
```

Comment:
PS. `dotnet nuke TargetNameHere` works as well and this is what I use from commandline

Reply:
Thinking about changing cmd line to: 
`dotnet nuke [--target] TargetNameHere`

Wondering if it's understandable?

Reply:
🤷 For me the output of `dotnet nuke help` is enough 😉

Reply:
Since it is "TargetNameHere" I think that the optional parameter doesn't add value.